src/Lykke.AzureStorage.Test/Lykke.AzureStorage.Test.csproj

```diff
@@ -23,10 +23,4 @@
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```

Comment:
Please update the nugets. By default tests fail due to https://developercommunity.visualstudio.com/content/problem/14673/unit-test-fail-cannot-load-microsoftvisualstudiote.html

Reply:
Test nugets is updated